### PR TITLE
Remove webconsole 3.x config

### PR DESCRIPTION
**Pull request by benjaminapetersen**
We no longer use this config, it has been replaced by top-level console & operator config.

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
@openshift/api-reviewers

Is there any reason to not remove what we no longer use?

File: webconsole/install.go [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-
-import (
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	webconsolev1 "github.com/openshift/api/webconsole/v1"
-)
-
-const (
-	GroupName = "webconsole.openshift.io"
```
Comment (soltysh) [inline webconsole/install.go]: We can't remove stuff just like that b/c we don't use it. We need to deprecate it first and follow the usual guidelines. Upstream has 3 releases or 1 year whichever is longer, not sure what about us? /cc @deads2k

Comment (benjaminapetersen) [reply webconsole/install.go]: Thanks, that's definitely why I opened this.  We don't use it, so it should at least be deprecated and planned to remove in the future if possible.

**Comment by deads2k** [comment]
/lgtm

This was only used in oc cluster up, we're good to remove

**Comment by spadgett** [comment]
/hold
to prevent retest until errors are fixed

**Comment by benjaminapetersen** [comment]
Yeah thats a little noisy.

**Comment by soltysh** [comment]
@benjaminapetersen can you refresh this and lets get it merged this week?

File: webconsole/install.go (hunk 1 above)
Comment (benjaminapetersen) [reply webconsole/install.go]: Resolving per @deads2k comment confirming that we can remove.

**Comment by benjaminapetersen** [comment]
Rebased, fixed `install.go`, `go build .` runs now, should be good.

File: install.go [hunk 2]
```diff
@@ -77,7 +76,6 @@ var (
 		servicecertsigner.Install,
 		template.Install,
 		user.Install,
-		webconsole.Install,
```
Comment (benjaminapetersen) [inline install.go]: Note that our new `/console` with our `Extensions` `CRD`s is not listed here.

**Pull request merged**